packages/css/src/plugins/sort-shorthand-declarations.ts

```diff
@@ -12,14 +12,6 @@ const findDeclaration = (node: ChildNode): Declaration | Declaration[] | undefin
     if (node.nodes.length === 1 && nodeIsDeclaration(node.nodes[0])) {
       return node.nodes[0];
     }
-
-    const declarations = node.nodes.map(findDeclaration).filter(Boolean) as Declaration[];
-
-    if (declarations.length === 1) {
```

Comment:
I deleted this as it would cause
```
@media (max-width: 100px) { ._abcd1234 { border-top: 1px solid } }
```
to be sorted with
```
@media (max-width: 100px) { ._abcd1234 { border: 1px solid } }
```

which overrides the sorting from the `sort-at-rules` plugin and I don't think is intentional?

Reply:
If I had to guess, I think I had issues understanding what sorting even looked at in the beginning, eg. there's a lot of different types of styles:
 - a whole stylesheet with multiple classes, media, etc in it
 - an `@media` query with multiple classes in it
 - a single `.class {property}`
 - A multiple `.class {property, webkitproperty}` — though I don't know if I even solved for that.

…so yeah, treat any of the code (if I wrote this) around here as possibly wrong!